more_itertools/more.py

```diff
@@ -573,6 +572,9 @@ def one(iterable, too_short=None, too_long=None):
         msg = (
             f'Expected exactly one item in iterable, but got {first_value!r}, '
             f'{second_value!r}, and perhaps more.'
+            f'Expected exactly one item in iterable, '
+            f'but got {first_value!r}, {second_value!r}, '
+            'and perhaps more.'
```

Comment:
Could we try to understand why these lines are being duplicated? Was this the result of `ruff` running, or some manual tweaking?

Reply:
No, but I'll play with this and check which variant is less multi-lined.